src/app/status-badge/status-badge.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { IncidentType } from '../types/statuspal';
+import { StatusService } from '../store/status.service';
+
+@Component({
+  selector: 'app-status-badge',
+  templateUrl: './status-badge.component.html',
+  styleUrls: ['./status-badge.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class StatusBadgeComponent implements OnInit {
+  status?: string;
+
+  constructor(private statusService: StatusService, private changeDetectorRef: ChangeDetectorRef) {}
+
+  ngOnInit(): void {
+    this.loadStatus();
+  }
+
+  openStatusPage(): void {
+    window.open('https://status.appuio.cloud', '_blank')?.focus();
```

Comment:
I always like when I know where a link points to. If I could wish, I'd use a plain old <a href ... link then, the user knows where the button takes him/her.